config/locales/de/models.yml

```diff
@@ -308,21 +308,21 @@ de:
           create: "Erstellt"
           destroy: "Gelöscht"
 
-      client: # TODO: Translate
+      client:
         label: "Label"
         config: "Konfiguration"
         domain: "Domain"
         secret_key: "Geheime API-Schlüssel"
         public_key: "Public API-Schlüssel"
         enabled: "Freigegeben"
-        location: "Canonical Location"
+        location: "autorisierte Stelle"
```

Comment:
This is a title, it should be capitalised.

Reply:
"Autorisierte Stelle"